node-js-basics-101/node_modules/ws/lib/Constants.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const safeBuffer = require('safe-buffer');
```

Comment:
__[jshint]__
* 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->